dev-utils/src/commands/test_vector_command.rs

```diff
@@ -0,0 +1,595 @@
+use abci::*;
+use bit_vec::BitVec;
+use kvdb_memorydb::create;
+use parity_scale_codec::Encode;
+use secp256k1::key::{PublicKey, SecretKey, XOnlyPublicKey};
+use secp256k1::schnorrsig::schnorr_sign;
+use secp256k1::{All, Message, Secp256k1, Signing};
+use serde::Serialize;
+use std::collections::BTreeMap;
+use std::convert::TryInto;
+use std::str::FromStr;
+use std::sync::Arc;
+
+use chain_abci::app::{BufferType, ChainNodeApp};
+use chain_abci::enclave_bridge::mock::MockClient;
+use chain_core::common::MerkleTree;
+use chain_core::compute_app_hash;
+use chain_core::init::address::{CroAddress, RedeemAddress};
+use chain_core::init::coin::Coin;
+use chain_core::init::config::InitConfig;
+use chain_core::init::network::Network;
+use chain_core::init::params::{
+    InitNetworkParameters, JailingParameters, NetworkParameters, RewardsParameters, SlashRatio,
+    SlashingParameters,
+};
+use chain_core::state::account::{
+    ConfidentialInit, CouncilNode, DepositBondTx, StakedState, StakedStateAddress,
+    StakedStateDestination, StakedStateOpAttributes, StakedStateOpWitness, UnbondTx,
+    WithdrawUnbondedTx,
+};
+use chain_core::state::tendermint::{TendermintValidatorPubKey, TendermintVotePower};
+use chain_core::state::validator::NodeJoinRequestTx;
+use chain_core::tx::data::access::{TxAccess, TxAccessPolicy};
+use chain_core::tx::data::address::ExtendedAddr;
+use chain_core::tx::data::attribute::TxAttributes;
+use chain_core::tx::data::input::{TxoPointer, TxoSize};
+use chain_core::tx::data::output::TxOut;
+use chain_core::tx::data::{Tx, TxId};
+use chain_core::tx::fee::{LinearFee, Milli};
+use chain_core::tx::witness::tree::RawXOnlyPubkey;
+use chain_core::tx::witness::{EcdsaSignature, TxInWitness, TxWitness};
+use chain_core::tx::TransactionId;
+use chain_core::tx::{PlainTxAux, TxAux, TxEnclaveAux, TxPublicAux};
+use chain_storage::buffer::Get;
+use chain_storage::{LookupItem, Storage, NUM_COLUMNS};
+use client_common::Result;
+use mock_utils::encrypt;
```

Comment:
The `mock_utils` is only used for development conveniences on non-sgx platforms, the obfustication result is different from the real platform.

Reply:
yes, I'll try to use the real one.